### PR TITLE
Added a Title Menu Scene with a Sceneloading script

### PR DESCRIPTION
**Pull request by indikman**
Added a new scene for the title menu, including a button as a prefab which can be modified later on. Also added a small script to handle scene loads. This can also be used to manage the scene loading animation / fades to load the scenes.

### CODE REVIEW

**Comment by indikman** [comment]
> [CLA assistant check](https://cla-assistant.io/UnityTechnologies/open-project-1?pullRequest=47)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/UnityTechnologies/open-project-1?pullRequest=47) before we can accept your contribution.
> 
> **Indika Wijesooriya** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/UnityTechnologies/open-project-1?pullRequest=47) it.

My Email is added and the CLA is already signed. Not sure why this message keeps coming.

File: UOP1_Project/ProjectSettings/EditorBuildSettings.asset [hunk 1]
```diff
@@ -5,7 +5,10 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
+  - enabled: 0
+    path: 
+    guid: 00000000000000000000000000000000
   - enabled: 1
-    path: Assets/Scenes/SampleScene.unity
-    guid: d1c3109bdb54ad54c8a2b2838528e640
+    path: Assets/Scenes/TitleMenu.unity
```
Comment (HyagoOliveira) [inline UOP1_Project/ProjectSettings/EditorBuildSettings.asset:12]: You shouldn't submit this file. Please remove from your PR.

File: UOP1_Project/ProjectSettings/QualitySettings.asset [hunk 2]
```diff
@@ -21,7 +21,7 @@ QualitySettings:
     skinWeights: 2
     textureQuality: 0
     anisotropicTextures: 1
-    antiAliasing: 2
+    antiAliasing: 0
```
Comment (HyagoOliveira) [inline UOP1_Project/ProjectSettings/QualitySettings.asset:24]: You shouldn't submit this file. Please remove from your PR.

**Review by HyagoOliveira — CHANGES_REQUESTED** [review]

File: UOP1_Project/ProjectSettings/EditorBuildSettings.asset (hunk 1 above)
Comment (indikman) [reply UOP1_Project/ProjectSettings/EditorBuildSettings.asset:12]: Hi Cant we add new Scenes? What is the reason for the request?

**Pull request closed without merging**